src/data/committee.js

```diff
@@ -101,6 +101,14 @@ const committee = {
           ai_concept: 'Generative AI.',
           fact: 'I like taking photos too!',
         },
+        {
+          name: 'Emil Medzhidov',
+          position: 'Development officer',
+          image: 'emil.medzhidov.jpg',
+          course: 'BEng Chemical Engineering (1st Year)',
+          ai_concept: 'Robotics',

```

Comment:
Hey, please add some sort of punctuation at the end, maybe an emoji? 🤖

Reply:
Sure, the changes have been added!